### PR TITLE
Fixed incorrect documentation link in admin UI

### LINKED ISSUE #5609: [0.10.0-1] [http] Link to documentation points to 0.9
The link in the InfluxDB web gui is still set to use: https://influxdb.com/docs/v0.9/introduction/getting_started.html

### PR DESCRIPTION
**Pull request by rossmcdonald**
Fixes #5609

### CODE REVIEW

**Comment by e-dard** [comment]
GH won't let me see the diff as the line is too long. I trust ya @rossmcdonald 

:+1:

**Comment by rossmcdonald** [comment]
@e-dard Hah, well your trust may have been misplaced. I think I have it fixed now, though. 

I wasn't aware that the admin UI needs to be regenerated with the `statik` tool on every change, so my original edits didn't have any effect. I also fixed all of the incorrect doc links I could find, which should hopefully be all of them.

@mjdesa It looks like you've regenerated the admin UI in the past. Can you double check [this readme I addded](https://github.com/influxdata/influxdb/blob/e92d502562be79c749c154578622230edb559156/statik/README.md) make sure I didn't do anything glaringly wrong?

File: importer/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ http://get.influxdb.org.s3.amazonaws.com/influxdb-0.8.9-1.x86_64.rpm
 
 `0.8.9` exports raw data to a flat file that includes two sections, `DDL` and `DML`.  You can choose to export them independently (see below).
 
-The `DDL` section contains the sql commands to create databases and retention policies.  the `DML` section is [line protocol](https://github.com/influxdb/influxdb/blob/master/tsdb/README.md) and can be directly posted to the [http endpoint](https://influxdb.com/docs/v0.9/guides/writing_data.html) in `0.9`.  Remember that batching is important and we don't recommend batch sizes over 5k.
+The `DDL` section contains the sql commands to create databases and retention policies.  the `DML` section is [line protocol](https://github.com/influxdb/influxdb/blob/master/tsdb/README.md) and can be directly posted to the [http endpoint](https://docs.influxdata.com/influxdb/v0.9/guides/writing_data) in `0.9`.  Remember that batching is important and we don't recommend batch sizes over 5k without further testing.
```
Comment (jwilder) [inline importer/README.md]: Should `v0.9` be `v0.10` now?

File: importer/README.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ curl -o export.dml.gz --compressed http://username:password@http://localhost:808
 
 ### Assumptions
 
-- Series name mapping follows these [guidelines](https://influxdb.com/docs/v0.8/advanced_topics/schema_design.html)
+- Series name mapping follows these [guidelines](https://docs.influxdata.com/influxdb/v0.8/advanced_topics/schema_design/)
 - Database name will map directly from `0.8` to `0.9`
```
Comment (jwilder) [inline importer/README.md]: `0.10` now?

File: importer/README.md (hunk 1 above)
Comment (rossmcdonald) [inline importer/README.md]: I wasn't sure if this was still the case for `0.10`, so I just updated the links. I can switch it to read `0.10` if it's still the same, though.

File: statik/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+### Regenerating the statik Admin UI
+
+If changes are made to the InfluxDB Admin UI, then this file can be
+regenerated using the following commands:
+
+```
+# Assuming the InfluxDB repo is located at `~/gocodes/src/github.com/influxdb/influxdb`
+go get github.com/rakyll/statik
+cd ~/gocodes/src/github.com/influxdb/influxdb
+~/gocodez/bin/statik -src=shared/admin
```
Comment (mjdesa) [inline statik/README.md]: you use `gocodez` and `gocodes`

File: importer/README.md (hunk 1 above)
Comment (rossmcdonald) [inline importer/README.md]: Fixed.

File: importer/README.md (hunk 2 above)
Comment (rossmcdonald) [inline importer/README.md]: Fixed.

File: statik/README.md (hunk 3 above)
Comment (rossmcdonald) [inline statik/README.md]: Ah, thanks. Fixed.

File: statik/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+[statik](https://github.com/rakyll/statik) tool, and should not be
+updated directly. It is generated from the contents of the
+`/shared/admin` directory, so make updates at that location first, and
+then continue with this guide.
+
+If changes are made to the InfluxDB Admin UI, then this file can be
+regenerated using the following commands:
+
+```
+# Assuming the InfluxDB repo is located at `~/gocodes/src/github.com/influxdb/influxdb`
```
Comment (gunnaraasen) [inline statik/README.md]: Probably want to use the `github.com/influxdata/influxdb` path in these examples.

File: importer/README.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ http://get.influxdb.org.s3.amazonaws.com/influxdb-0.8.9-1.x86_64.rpm
 
 `0.8.9` exports raw data to a flat file that includes two sections, `DDL` and `DML`.  You can choose to export them independently (see below).
 
-The `DDL` section contains the sql commands to create databases and retention policies.  the `DML` section is [line protocol](https://github.com/influxdb/influxdb/blob/master/tsdb/README.md) and can be directly posted to the [http endpoint](https://influxdb.com/docs/v0.9/guides/writing_data.html) in `0.9`.  Remember that batching is important and we don't recommend batch sizes over 5k.
+The `DDL` section contains the sql commands to create databases and retention policies.  the `DML` section is [line protocol](https://github.com/influxdb/influxdb/blob/master/tsdb/README.md) and can be directly posted to the [http endpoint](https://docs.influxdata.com/influxdb/v0.10/guides/writing_data) in `0.10`.  Remember that batching is important and we don't recommend batch sizes over 5k without further testing.
```
Comment (gunnaraasen) [inline importer/README.md]: Looks like this line is conflicting due to the recent path renames.

File: statik/README.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+go get github.com/rakyll/statik
+cd ~/gocodez/src/github.com/influxdb/influxdb
+~/gocodez/bin/statik -src=shared/admin
```
Comment (gunnaraasen) [inline statik/README.md]: Not sure if it's actually needed, but the [admin UI readme](https://github.com/influxdata/influxdb/blob/master/shared/admin/README.md) recommends running `go fmt` after this line, e.g. `go fmt statik/statik.go`.

Comment (rossmcdonald) [inline statik/README.md]: Mother... I didn't even see the admin readme, which is why I was confused. Okay, I'll just put a dummy readme here that references the admin one.

**Comment by rossmcdonald** [comment]
Okay, so:
- changed the statik readme to point to the admin readme
- all references to 0.9 have been fixed to point to 0.10
- updated paths with the switch to 'influxdata/influxdb' over 'influxdb/influxdb'

**Comment by e-dard** [comment]
Never again @rossmcdonald. Never again! Where did all this content come from, it was a single massive line this morning. 😂

**Comment by corylanou** [comment]
+1

**Comment by gunnaraasen** [comment]
+1 :shipit:

**Comment by toddboom** [comment]
:+1:

**Comment by toddboom** [comment]
@rossmcdonald Sorry, I hosed your ability to merge cleanly with another PR! ;)

**Comment by rossmcdonald** [comment]
***shakes fist angrily***

It's all good. I just had to rebuild the statik filesystem with the merged changes. I'll merge on green. Thanks for your help everyone!

**Pull request merged**